src/main/scala/firrtl_interpreter/vcd/VCD.scala

```diff
@@ -482,7 +482,9 @@ case class VCD(
   val initialValues = new mutable.HashSet[Change]
   var scopeRoot = Scope(scope)
   val wires = new mutable.HashMap[String, Wire]
-  var aliasedWires = new mutable.HashMap[String, mutable.HashSet[Wire]] {
+  var aliasedWires : mutable.HashMap[String, mutable.HashSet[Wire]] =
```

Comment:
Did IntelliJ suggest this change? It looks like a DRY-violation to me.

Reply:
yes it did, it's quite particular about public members of classes having explicit declarations.